queries.sql

```diff
@@ -0,0 +1,25 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM vet_clinic
+WHERE name LIKE '%mon';
+
+SELECT name FROM vet_clinic
+WHERE date_of_birth between '2016-01-01' and '2019-01-01';
```

Comment:
- [ ] This query suppose to list the name of all animals born between the start of 2016 to the end of 2019, not like you did.
[Hint] this is how it's supposed to be:
```sql
SELECT name FROM animals WHERE date_of_birth between '2016-01-01' and '2019-31-12';

```